tests/analyses/decompiler/test_decompiler.py

```diff
@@ -525,7 +525,11 @@ def test_decompiling_1after909_verify_password(self, decompiler_options=None):
 
     @for_all_structuring_algos
     def test_decompiling_1after909_doit(self, decompiler_options=None):
-        # the doit() function has an abnormal loop at 0x1d47 - 0x1da1 - 0x1d73
+        """
+        The doit() function has an abnormal loop at 0x1d47 - 0x1da1 - 0x1d73.
+        The original source code can be found here:
+        https://github.com/shellphish/ictf20-challenges/blob/1e0b7c1fde9b5c8ff2d3e1ca428c4396d63e046e/1after909/src/1after909.c#L298
+        """
```

Comment:
I believe doc strings in test cases will be used in the CI output instead of the function name. That's why we have been avoiding having doc strings for test cases. Can you check if it's still the case with CI?

Reply:
@ltfish how can I check this? I've been adding docstrings into the tests for a while now... and I don't see them in the output at least on GitHub.

Reply:
Yeah I dont see it used in the CI